source/stylesheets/refills/_vertical-tabs.scss

```diff
@@ -1,51 +1,36 @@
 .vertical-tabs-container {
-  ///////////////////////////////////////////////////////////////////////////////////
-  $base-border-color: gainsboro !default;
-  $base-border-radius: 3px !default;
-  $base-background-color: white !default;
-  $base-line-height: 1.5em !default;
-  $base-spacing: 1.5em !default;
-  $base-accent-color: #477DCA !default;
-  $base-link-color: $base-accent-color !default;
-  $dark-gray: #333 !default;
-  $medium-screen: em(640) !default;
-  $base-font-color: $dark-gray !default;
-  
-  li {
-    list-style: none;
-  }
-
-  p {
-    color: $base-font-color;
-    line-height: $base-line-height;
-  }
-
-  a {
-    text-decoration: none;
-    color: $base-link-color;
-  }
-  //////////////////////////////////////////////////////////////////////////////////
-
+  $tab-color: $base-font-color-on-box;
+  $tab-color-inactive: $base-font-color-on-box;
   $tab-border-color: $base-border-color;
-  $tab-border: 1px solid $tab-border-color;
+  $tab-border: $base-border;
   $tab-border-radius: $base-border-radius;
-  $tab-content-background: lighten($tab-border-color, 10%);
-  $tab-active-color: $tab-content-background;
-  $tab-inactive-color: $base-background-color;
+  $tab-content-background: $base-box-background;
+  $tab-active-background: $tab-content-background;
+  $tab-inactive-background: $base-box-background-inset;
   $vertical-tabs-height: em(300);
   $vertical-tab-mode: $medium-screen;
 
   @include clearfix;
-  border-radius: $tab-border-radius;
+  color: $tab-color;
```

Comment:
Properties should be ordered border, border-radius, color, margin-bottom, overflow